### PR TITLE
chore: Use the latest backup.

### PR DESCRIPTION
**Pull request by Blu-J**

### CODE REVIEW

**Comment by Blu-J** [comment]
Pushed up an alpha image

File: manifest.json [hunk 1]
```diff
@@ -1,17 +1,15 @@
 {
   "id": "vaultwarden",
   "title": "Vaultwarden",
-  "version": "1.27.0",
-  "release-notes": "* Update upstream to v1.27.0, official release notes: https://github.com/dani-garcia/vaultwarden/releases/tag/1.27.0 \n * Fix a bug with client connections",
+  "version": "1.27.0.1",
+  "release-notes": "* Using js for the backups instead of emulated python",
```
Comment (elvece) [inline manifest.json:5]:
```suggestion
  "release-notes": "* Use new eOS APIs for backups",
```

**Review by elvece — APPROVED** [review]
suggestion for release notes

**Comment by Blu-J** [comment]
Tested that the backup works with my nucpi + usb backup

**Pull request merged**